src/meteofrance/helpers.py

```diff
@@ -16,7 +18,9 @@
 from .model.place import Place
 
 
-def get_warning_text_status_from_indice_color(int_color: int, lang: str = "fr") -> str:
+def get_warning_text_status_from_indice_color(
+    int_color: int, lang: str = "fr"
+) -> Optional[str]:
```

Comment:
remove optional

Reply:
Currently optional is needed due to the fact that first element of ALERT_COLOR_LIST_FR and ALERT_COLOR_LIST_EN is None.
Another way could be to check if 0 < int_color < 5 and raise a ValueError or IndexError exception if False.